pkg/core/helper_test.go

```diff
@@ -279,16 +279,34 @@ func initBasicChain(t *testing.T, bc *Blockchain) {
 
 	priv0 := testchain.PrivateKeyByID(0)
 	priv0ScriptHash := priv0.GetScriptHash()
+	priv1 := testchain.PrivateKeyByID(1)
+	priv1ScriptHash := priv1.GetScriptHash()
+	acc0 := wallet.NewAccountFromPrivateKey(priv0)
+	acc1 := wallet.NewAccountFromPrivateKey(priv1)
+
+	deployContractFromPriv0 := func(t *testing.T, path, contractName string, configPath *string, expectedID int32) (util.Uint256, util.Uint256, util.Uint160) {
```

Comment:
Can `initBasicChain` be done via our testing framework (just asking!)?

Reply:
I think yes, create an issue?

Reply:
Yeah, if there are no known blockers it'd make things much easier to maintain.

Reply:
https://github.com/nspcc-dev/neo-go/issues/2355